privacyidea/lib/token.py

```diff
@@ -1972,7 +1973,8 @@ def check_realm_pass(realm, passw, options=None,
             return False, reply_dict
 
         return check_token_list(tokenobject_list, passw, options=options,
-                                allow_reset_all_tokens=False)
+                                allow_reset_all_tokens=False,
+                                allow_increase_failcount=False)
```

Comment:
Actually you do not need to add "False", if it is False by default anyways

Reply:
fixed in 4fe9f7f46e86b88dcd365672f86288d621345875.